python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/dot1x.j2

```diff
@@ -108,6 +108,43 @@ dot1x
    captive-portal start limit infinite
 {%             endif %}
 {%         endif %}
+{%         if dot1x.radius_av_pair.lldp is arista.avd.defined or dot1x.radius_av_pair.dhcp is arista.avd.defined %}
```

Comment:
we can keep separate defined check for both as they are in sequence separately

Reply:
updated